src/test/e2e/helpers/test-helper.ts

```diff
@@ -357,6 +358,7 @@ async function createApp(
         },
     });
     const services = createServices(stores, config, db);
+    await initialServiceSetup(config, services);
```

Comment:
I wonder if we should have an option that controls whether we want to initialize the services here, for those cases where we want to specifically test the init logic so we want to trigger it explicitly.

Not important for now though.

Reply:
I think if this is the case we're probably not calling `createApp` method which is more for e2e tests, but yes, this allows us to initialize when we want rather than having a process.nextTick initialize as a side effect